### ISSUE TITLE
Accessibility: Tables not navigable by keyboard

### ISSUE THREAD

**Issue by NicoleGovvy** [bcgov/cas-registration#3027]
#### Describe the Bug: Keyboard navigation fails on pages with tables. The cursor stops at the last search field and is unable to access any of the links in the actions column. This prevents those reliant on keyboard-only navigation from being able to access key information and also is a keyboard trap – the user cannot proceed any further in the process.

Tables needing to be updated:

Industry side [NOTE all tables in Test as of March 17, 2025]
1. Administration > Operations table
2. Administration > Contacts table
3. Reporting > Submit annual reports

CAS side [NOTE all tables in Test as of March 17, 2025]
1. Administration > Operators table
2. Administration > Operations table
3. Administration > Operator Administrators And Access Requests table
4. Administration > Contacts table
5. Registration > Transfers table

#### Probability: 5

(How likely the bug is to happen, scored from 1-5. For example, probability of 5 is something like "it happens to all users every time they log in." whereas probability of 1 "only happens to certain users when a really specific and unlikely path is followed.")

#### Impact:

5 for individuals relying on keyboard navigation as they are unable to interact with any links within a table or navigate beyond the table search boxes.

<img src="https://www.getzephyr.com/sites/default/files/riksk%20matrix.jpg" alt="Bug Matrix" width="250" height="250"/>

#### Steps to reproduce the behaviour:

1. Open BCIERS
2. Open 'Submit a report' [example, all tables have this issue]
3. Try to navigate with a keyboard through the screen.

#### Screenshots:

![Image](https://github.com/user-attachments/assets/e25f8d44-aa8e-461e-b2de-c89cbe4f0b4c)

#### Additional information:

Browser: [e.g. chrome, safari]
Version: [e.g. 22]

#### Additional Context

-
